.codeclimate.yml

```diff
@@ -10,7 +10,7 @@ engines:
   rubocop:
     enabled: true
   bundler-audit:
-    enabled: true
+    enabled: false
```

Comment:
According to this: https://codeclimate.com/github/sendgrid/sendgrid-ruby/pull/281

It seems the problem is that we don't have a Gemfile.lock file. It seems like that would be the correct fix.

Thoughts?

Reply:
@thinkingserious As far as I know it is not recommended to keep Gemfile.lock in repository when developing gem
https://github.com/sendgrid/sendgrid-ruby/blob/master/.gitignore#L34-L36
http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

As you can see in [sendgrid-ruby.gemspec](https://github.com/sendgrid/sendgrid-ruby/blob/master/sendgrid-ruby.gemspec) only gems constraints are provided, so every time when somebody installs sendgrid-ruby gem, she/he gets the versions of dependencies suitable to context of installation (whether it's done by `bundle install` or `gem install` different set of versions might be installed).

I'm not sure if it makes sense to perform bundler audit check.

Reply:
Thanks @szemek! I appreciate the quick response.

That makes sense.

Reply:
@thinkingserious do you think my changes can be merged, do I need to improve anything or should I close my PR?

Reply:
@szemek,

Yes, I'm just waiting for this issue to hit the top of my backlog. Thanks!